src/decider/agent/DeciderAgent.php

```diff
@@ -543,4 +561,89 @@ protected function areOtherScannerFindingsAndRESOAgreed($licenseMatches)
     }
     return true;
   }
-}
+
+  private function getCopyrightsToDisableFalsePositives($uploadId)
+  {
+    if (!$uploadId) {
+      return array();
+    }
+    $agentName = 'copyright';
+    $uploadTreeTableName = $this->uploadDao->getUploadtreeTableName($uploadId);
+    $scanJobProxy = new ScanJobProxy($GLOBALS['container']->get('dao.agent'), $uploadId);
+    $scanJobProxy->createAgentStatus(array($agentName));
+    $selectedScanners = $scanJobProxy->getLatestSuccessfulAgentIds();
+    if (!array_key_exists($agentName, $selectedScanners)) {
+      return array();
+    }
+    $latestXpAgentId = $selectedScanners[$agentName];
+    $extrawhere = ' agent_fk='.$latestXpAgentId;
+    $allCopyrights = $this->copyrightDao->getScannerEntries('copyright',
+      $uploadTreeTableName, $uploadId, null, $extrawhere);
+
+    $copyrightJSON = json_encode($allCopyrights);
+    $tmpFile = tmpfile();
+    $tmpFilePath = stream_get_meta_data($tmpFile)['uri'];
+    fwrite($tmpFile, $copyrightJSON);
+    $deactivatedCopyrightData = $this->callCopyrightDeactivationClutterRemovalScript($tmpFilePath);
+    
+    if(!$deactivatedCopyrightData) {
+      return array();
+    }
+    $deactivatedCopyrights = json_decode($deactivatedCopyrightData, true);
+    foreach ($deactivatedCopyrights as $deactivatedCopyright) {
+      $item = $deactivatedCopyright['uploadtree_pk'];
+      $hash = $deactivatedCopyright['hash'];
+      $content = $deactivatedCopyright['content'];
+      $cpTable = 'copyright';
+      
+      if ($deactivatedCopyright['is_copyright'] == "t"){
+        $action = ' ';
+      }
+      else {
+        $action = 'delete';
+      }
+      $this->copyrightDao->updateTable($item, $hash, $content, $this->userId, $cpTable, $action);
+    }
+    // $handle = fopen("/home/polymath/Desktop/fossology/handleLog.txt", "a");
+    // fwrite($handle, print_r($deactivatedCopyrightData, true));
+    // fclose($handle);
+
+  }
+  private function getCopyrightsToDisableFalsePositivesClutter($uploadId)
+  {
+    if (!$uploadId) {
+      return [];
+    }
+    $uploadTreeTableName = $this->uploadDao->getUploadtreeTableName($uploadId);
+    $allCopyrights = $this->copyrightDao->getScannerEntries('copyright',
+      $uploadTreeTableName, $uploadId, null, null);
+    
+    $handle = fopen("/home/polymath/Desktop/fossology/handleLog.txt", "a");
+    fwrite($handle, print_r($allCopyrights, true));
+    fclose($handle);
+    
+    foreach ($allCopyrights as $finding) {
+      $filesWithLicenses[$finding['uploadtree_pk']]['copyrights'][] = \convertToUTF8($finding['content'],false);
+    }
+  }
+  
+  private function callCopyrightDeactivationClutterRemovalScript($tmpFilePath, $clutter_flag = 0) 
+  {
+    $script = "copyrightDeactivationClutterRemovalScript.py";
+    $path_to_file = "/home/polymath/Desktop/fossology/src/decider/agent/copyrightDeactivationClutterRemovalScript.py";
+    // __DIR__ . "/$script";
```

Comment:
Please modify the `Makefile` to install the script as well here: https://github.com/fossology/fossology/blob/62d61b1f95fa1e18614b82097b37ab81c62702f5/src/decider/agent/Makefile#L33-L42

`$(INSTALL_PROGRAM) $(SCRIPT) $(DESTDIR)$(MODDIR)/$(MODNAME)/agent/$(SCRIPT)`

This will allow to use `__DIR__`